### ISSUE TITLE
Can't configure Token on NixOS

### ISSUE THREAD

**Issue by squiter** [sachaos/todoist#165]
Hey folks, I can't configure the cli to use the Token when using in NixOS.

I created the config file with my token:

```
$ cat ~/.config/todoist/config.json
{
  "token": "my-token", # todoist api token, required
  "color": "true"                                      # colorize all output, not required, default false
}
```

And when I run the `sync` command I can see that my token isn't setted up correctly:

```
$ todoist --debug sync
2021/08/31 17:37:57 doAPi: called
2021/08/31 17:37:57 config: &todoist.Config{AccessToken:"", DebugMode:true, Color:false}
2021/08/31 17:37:57 request: &http.Request{Method:"POST", URL:(*url.URL)(0xc000155050), Proto:"HTTP/1.1", ProtoMajor:1, ProtoMinor:1, Header:http.Header{"Content-Type":[]string{"application/x-www-form-urlencoded"}}, Body:io.nopCloser{Reader:(*strings.Reader)(0xc0000d4920)}, GetBody:(func() (io.ReadCloser, error))(0x6a5080), ContentLength:52, TransferEncoding:[]string(nil), Close:false, Host:"todoist.com", Form:url.Values(nil), PostForm:url.Values(nil), MultipartForm:(*multipart.Form)(nil), Trailer:http.Header(nil), RemoteAddr:"", RequestURI:"", TLS:(*tls.ConnectionState)(nil), Cancel:(<-chan struct {})(nil), Response:(*http.Response)(nil), ctx:(*context.emptyCtx)(0xc0000b4000)}
2021/08/31 17:37:57 request.URL: &url.URL{Scheme:"https", Opaque:"", User:(*url.Userinfo)(nil), Host:"todoist.com", Path:"/API/v8/sync", RawPath:"", ForceQuery:false, RawQuery:"", Fragment:"", RawFragment:""}
2021/08/31 17:37:57 params: &strings.Reader{s:"resource_types=%5B%22all%22%5D&sync_token=%2A&token=", i:0, prevRune:-1}
2021/08/31 17:37:57 response: &http.Response{Status:"403 Forbidden", StatusCode:403, Proto:"HTTP/2.0", ProtoMajor:2, ProtoMinor:0, Header:http.Header{"Cache-Control":[]string{"no-cache"}, "Content-Length":[]string{"190"}, "Content-Type":[]string{"application/json"}, "Date":[]string{"Tue, 31 Aug 2021 20:37:57 GMT"}, "Referrer-Policy":[]string{"strict-origin-when-cross-origin"}, "Retry-After":[]string{"4"}, "Strict-Transport-Security":[]string{"max-age=31536000; includeSubDomains; preload"}, "Via":[]string{"1.1 9a0470365001a35d1ab4fa1b7a9f9daf.cloudfront.net (CloudFront)"}, "X-Amz-Cf-Id":[]string{"1c50_GkhlrYXv3Yyp82EL6jFFOPf608q313ToqMTvmsOcalxEG04uA=="}, "X-Amz-Cf-Pop":[]string{"GRU3-C1"}, "X-Cache":[]string{"Error from cloudfront"}}, Body:http.http2transportResponseBody{cs:(*http.http2clientStream)(0xc00022e580)}, ContentLength:190, TransferEncoding:[]string(nil), Close:false, Uncompressed:false, Trailer:http.Header(nil), Request:(*http.Request)(0xc000118400), TLS:(*tls.ConnectionState)(0xc000439340)}
2021/08/31 17:37:57 bad request: 403 Forbidden: Invalid token
Error: bad request: 403 Forbidden
```

It seems the cli isn't check in the default directory :thinking: 

Can you help me to figure out what I did wrong?

**Comment by timtraversy** [comment]
Hey - I hit the same issue on NixOS. After grepping, I found the config file under `~/.todoist.config.json`, instead of what the README indicates. I can't tell exactly why this happens from the source code, but I hope this helps.

**Comment by sagikazarmark** [comment]
Is there a reason why the config file has to be in the home directory on NixOS?

**Comment by sagikazarmark** [comment]
Oh, I guess because #23 was implemented in July 2020, whereas the latest release is from January 2020.

@sachaos do you think you could tag a new release?

**Comment by sachaos** [comment]
@sagikazarmark Thank you remind me. I've released v0.16.0 https://github.com/sachaos/todoist/releases/tag/v0.16.0

**Comment by sagikazarmark** [comment]
@sachaos Thanks!

Although it doesn't affect functionality, but the version was not updated: https://github.com/sachaos/todoist/blob/master/main.go#L46

Maybe it would make sense to inject that during the release process using ldflags?
